pkg/api/norman/customization/cluster/validator.go

```diff
@@ -668,6 +669,16 @@ func validateGKENodePools(spec *v32.ClusterSpec) error {
 			errors = append(errors, fmt.Sprintf("node pool [%s] version cannot be empty", name))
 			continue
 		}
+
+		// Windows images are WINDOWS_LTSC or WINDOWS_SAC. The cluster must have at least one non-Windows node pool.
+		image := strings.ToLower(np.Config.ImageType)
+		if !hasRequiredLinuxPool && !strings.Contains(image, "windows") {
+			hasRequiredLinuxPool = true
+		}
```

Comment:
nit: not necessary to have the `image` variable

Reply:
done